x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/constants.tsx

```diff
@@ -6,13 +6,15 @@
  */
 
 export const DATE_FIELD_TYPE = 'date';
+export const EVENT_MODULE_FIELD_NAME = 'event.module';
+export const EVENT_URL_FIELD_NAME = 'event.url';
+export const GEO_FIELD_TYPE = 'geo_point';
 export const HOST_NAME_FIELD_NAME = 'host.name';
 export const IP_FIELD_TYPE = 'ip';
-export const GEO_FIELD_TYPE = 'geo_point';
 export const MESSAGE_FIELD_NAME = 'message';
-export const EVENT_MODULE_FIELD_NAME = 'event.module';
-export const RULE_REFERENCE_FIELD_NAME = 'rule.reference';
 export const REFERENCE_URL_FIELD_NAME = 'reference.url';
-export const EVENT_URL_FIELD_NAME = 'event.url';
+export const RULE_REFERENCE_FIELD_NAME = 'rule.reference';
 export const SIGNAL_RULE_NAME_FIELD_NAME = 'signal.rule.name';
 export const SIGNAL_STATUS_FIELD_NAME = 'signal.status';
+export const THREAT_INDICATOR_EVENT_URL_FIELD_NAME = 'threat.indicator.event.url';
```

Comment:
Do you think I should remove these on my row renderer PR? I've got them in `common/cti/constants` so we could certainly consolidate this.

Reply:
consolidating them is a great idea, I can create a mini chore ticket to tackle this once both of our PRs are merged, I am currently killing this one for an updated implementation as a part of #998